portfolio/src/main/webapp/script.js

```diff
@@ -12,19 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random fact about me to the page.
- */
-
+/** Adds a random fact about me to the page.*/
 function addRandomFact() {
   const facts = ['I love tropical destinations and spending time by the ocean!', 'I have torn the ACLs in both my knees playing soccer (at separate times). I got surgery to repair both of them.', 'I am a huge cat person (but I love most pets too).', 'I am a huge foodie. My favourite foods are sushi, thai basil beef and tacos.',
   'I have a huge sweet tooth for desserts, especially any homemade baked good.', 'I\m a pretty decent mariokart player.', 'I love being creative. I like drawing, painting, and any form of art.'];
 
   // Pick a random fact.
- const fact = facts[Math.floor(Math.random() * facts.length)];
-
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+/** Fetches tasks from the server and adds them to the DOM. */
+function loadTasks() {
+  // get user selection for quantity
+  getSelect(); 
+
+  fetch('/data').then(response => response.json()).then((tasks) => {
```

Comment:
Since we're probably going to switch to allow this to update live, we'll want to clear the DOM before repopulating it.